projects/imu_driver/inc/imu.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+// Imu module written for IMU board
+// This module is for communicating accelerometer and gyroscope data
+// from the car to the telemetry server for strategic purposes.
+#include "gpio.h"
+#include "spi.h"
+
+typedef enum gyro_read_axes { GYRO_X = 0, GYRO_Y, GYRO_Z } gyro_read_axes;
```

Comment:
We normally want the enum on separate lines, also in terms of naming, we generally want to prepend the module name (in this case `imu`). So maybe another name could be `imu_gyro_axes`. Also the last enum should generally be the number of enums (e.g. `NUM_IMU_GYRO_AXES`)